pkg/vpcmodel/explainabilityConnectivity.go

```diff
@@ -80,6 +80,10 @@ type Explanation struct {
 	// this information should be handy; otherwise empty (slice of size 0)
 	srcNetworkInterfacesFromIP []Node
 	dstNetworkInterfacesFromIP []Node
+	// the following two properties are for the case src/dst are given as nodeSet name
+	// this information should be handy; otherwise nil
+	srcNodeSet NodeSet
+	dstNodeSet NodeSet
```

Comment:
Perhaps define a struct for `srcNodeSet` and `dstNodeSet`; this will solve the lint issue and simplify the code in certain places. Also, maybe this is an opportunity to define a struct for  `srcNetworkInterfacesFromIP` and `dstNetworkInterfacesFromIP` as well.

Reply:
yes, after you reviw